src/Components/WebAssembly/Authentication.Msal/src/Models/MsalProviderOptions.cs

```diff
@@ -24,7 +24,12 @@ public class MsalProviderOptions
         /// <summary>
         /// Gets or sets the msal.js cache options.
         /// </summary>
-        public MsalCacheOptions Cache { get; set; }
+        public MsalCacheOptions Cache { get; set; } = new MsalCacheOptions
```

Comment:
Curious: why is setting the defaults moved to here and not in the MsalDefaultOptionsConfiguration?

Reply:
This way, the instance is already created when you try to dot into it in the setup option and it keeps the change in one place.

If you do it on PostConfigureOptions you need to be careful not to override the settings that the user might have set.

Reply:
Got it. Thanks for explaining. Resolving this.